### ISSUE TITLE
Error: file_browser.run: cannot open

### ISSUE THREAD

**Issue by The0zymandias** [awkitsune/IkuReader#1]
I installed the ikureader.nds entry from Universal Updater and am attempting to use it via Twilight Menu on my New 3DS. I extracted the sd card zip onto my sd card and it continues to not work. If you could look into this issue, I'd appreciate it.

**Comment by The0zymandias** [comment]
Ok i figured out a fix-ish. IkuReader only works with TwilightMenu if the IkuReader.nds file is on the root of the SD card. I have no idea why. This also works with nds forwarders, as long as the .nds file is on the root.

**Comment by awkitsune** [comment]
> Ok i figured out a fix-ish. IkuReader only works with TwilightMenu if the IkuReader.nds file is on the root of the SD card. I have no idea why. This also works with nds forwarders, as long as the .nds file is on the root.

Yep, I had this behavior too testing .nds version with TM, I was using .dsi file and installed it in DSi menu with NandTitleManager, this worked out fine. Probably it will work this way with 3DS too. 

Now when I got 3ds I will be able to test it with it and write a recommendations on usage. Thank you for your feedback! And if you have further questions feel free to write them ^_^

**Comment by The0zymandias** [comment]
> > Ok i figured out a fix-ish. IkuReader only works with TwilightMenu if the IkuReader.nds file is on the root of the SD card. I have no idea why. This also works with nds forwarders, as long as the .nds file is on the root.
> 
> Yep, I had this behavior too testing .nds version with TM, I was using .dsi file and installed it in DSi menu with NandTitleManager, this worked out fine. Probably it will work this way with 3DS too.
> 
> Now when I got 3ds I will be able to test it with it and write a recommendations on usage. Thank you for your feedback! And if you have further questions feel free to write them ^_^

If possible, when you get a 3DS, would you be able to make it so you can place the .nds file anywhere and it will still work? If you don’t know, TeilightMenu on the 3DS encrouages putting .nds roms in sdmc:/roms/nds and not on the root of the SD card.
